idea-fare/settings/base.py

```diff
@@ -138,9 +138,10 @@
 
 EMAIL = os.environ.get('EMAIL_USER', None)
 # Add '+contact' to email...for e.g source@example.com -> source+contact@example.com
-if EMAIL is not None: CONTACT_EMAIL = EMAIL.replace('@', '+contact@')
-# Source mail address used for notifications.
-COMMENTS_XTD_FROM_EMAIL = EMAIL
-# Contact mail address to show in messages.
-COMMENTS_XTD_CONTACT_EMAIL = CONTACT_EMAIL
+if EMAIL is not None: 
```

Comment:
This comment is misplaced. It should be a couple of lines below where it is placed currently. 
Also, there was an informative message when this variable is not set, it should be placed here.